src/content/docs/style-guide/punctuation.mdx

```diff
@@ -27,9 +27,13 @@ In writing, punctuation marks are used to imitate speech and play an important r
 -   Colons signal that what comes next is related directly to the previous word, phrase, or sentence.
 -   Never use a colon after a sentence fragment.
 
-### Examples
+General guidelines for using colons:
+
+##
 
--   Animal has 3 different styles for drumming: loud, louder, and deafening.
+-   In running text: Capitalize the first word after the colon if the word begins a complete sentence. Otherwise, do not capitalize the first word after a colon in running text.
+-   In headings: Capitalize the first word after the colon, regardless of its part of speech, unless the word is a product name that begins with a lowercase letter or, in developer materials, is a computer literal that begins with a lowercase letter.
+-   With a bullet list: Where there’s an introductory word or phrase (often bolded), capitalize the first word in the text that follows.
```

Comment:
@bhecknr is there value in specifying "lead-in phrases" and clarifying that this also applies to incomplete sentences?

In bullet list items with lead-ins: Where there’s an introductory word or phrase (often bolded), capitalize the first word in the text that follows&mdash;even if it's not a complete sentence.